File: rev-com-server/src/main/java/com/codergb/bean/Screenwriter.java
package com.codergb.bean;

import lombok.*;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
@EqualsAndHashCode(callSuper = true)
public class Screenwriter extends FileInfo{
  private String id;
  private String name;
  private String avatarUrl;
  private String alias;
  private String gender;
  private String birthPlace;
  private String profession;
  private String description;
  private Date createTime;
  private Date updateTime;
}

File: rev-com-manage/src/types/screenwriter.ts
import { IOccupation } from "@/types/occupation"

interface IScreenwriter {
  id: string
  name: string
  avatarUrl: string
  alias: string
  family: string
  occupations: IOccupation[]
  gender: string
  birthPlace: string
  birth: string
  description: string
  createTime: string
  updateTime: string
}
export type { IScreenwriter }
